pkg/controller/controller.go

```diff
@@ -24,3 +24,74 @@ func AddToManager(m manager.Manager) error {
 	}
 	return nil
 }
+
+func generateObjects() []runtime.Object {
+	// Construct the gvks for objects to watch.  Remove the Any
+	// kind or else all objects kinds will be watched.  That would
+	// be bad.
+	kubeLinterKinds := objectkinds.AllObjectKinds()
+	for i := range kubeLinterKinds {
+		if kubeLinterKinds[i] == objectkinds.Any {
+			kubeLinterKinds = append(kubeLinterKinds[:i], kubeLinterKinds[i+1:]...)
+			break
+		}
+	}
+
+	kubeLinterMatcher, err := objectkinds.ConstructMatcher(kubeLinterKinds...)
+	if err != nil {
+		// TODO log error or exit?
```

Comment:
what kind of errors will we have here?

Reply:
This should never return an error unless there is a problem in kube-linter itself afau